partiql-grammar/src/main/gen/PartiQLTokens.interp

```diff
@@ -0,0 +1,957 @@
+token literal names:
```

Comment:
BTW these files weren't supposed to be committed. @RCHowell moved the grammar to `:lang`.